sample/mpp-library/src/commonMain/kotlin/com/icerockdev/library/Testing.kt

```diff
@@ -33,7 +32,10 @@ object Testing {
             MR.plurals.test_plural.desc(1),
             MR.plurals.test_plural.desc(2),
             MR.plurals.test_plural.desc(3),
-            nestedTest()
+            MR.plurals.test_plural.desc(7),
+//            TODO nested won't work with the current implementation, since the library tries to resolve from a bundle
```

Comment:
I'm not sure how could we solve this

Reply:
plugin should collect all nested resources too. just like in apple generator implemented

Reply:
I'm not sure I know what you mean, currently resolving the nested will cause a 
```
Can't find resource for bundle java.util.PropertyResourceBundle, key nested.test
```

I think on android this works by default, because how android aggregates or searches through the sources, but I may be wrong.
I didn't research much on this, but maybe there is a similar solution for this on the jvm side too.
Did you meant copying over the generated resources from the nested modules, I'd consider that a bit overkill, but I'm not sure

Reply:
in uber-jar all resources aggregates into single jar, so if we have same name of some resources (like MokoBundle for localizations) we got only one version of this files. to fix it i will move generation of files into package-dependend path